getYear.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"errors"
+	"github.com/rwcarlsen/goexif/exif"
+	"log"
+	"os"
+)
+
+/**
+ * This function grabs the year from the file camera data
+ */
```

Comment:
This isn't a big deal, but the conventional way to comment functions in Go is this:
```suggestion
// getYearFromExif extracts the year from the file camera data and returns it.
```